03-specifying-sample-designs.Rmd

```diff
@@ -112,15 +114,15 @@ des_apisrs
 summary(des_apisrs)
 ```
 
-In the printed design object above, the design is described as an "Independent Sampling design," which is another terminology for SRS. The ids are specified as `1`, which means there is no clustering (a topic described later in this chapter), the fpc variable is indicated, and the weights are indicated. When looking at the summary of the design object, the population size is given as a summary of the probabilities (inverse of the weights).
+In the printed design object above, the design is described as an "Independent Sampling design," which is another terminology for SRS. The ids are specified as `1`, which means there is no clustering (a topic described later in this chapter), the fpc variable is indicated, and the weights are indicated. When looking at the summary of the design object, the population size is given as a summary of the probabilities (inverse of the weights). 
 
 ### Simple random sample with replacement
 
-- **Description**: The simple random sample with replacement (SRSWR) is a sampling design where a sample is selected from a sampling frame with the units being replaced before drawing again, so units can be selected more than once.
-- **Requirements**: The sampling frame must include the entire population.
-- **Advantages**: SRSWR requires no information about the units apart from contact information.
-- **Disadvantages**: The sampling frame may not be available for the entire population. This design is not generally feasible for in-person data collection. Units can be selected more than once, resulting in a smaller realized sample size. For small populations, SRSWR has larger standard errors than SRS designs.
-- **Example**: A professor puts all students' names on paper slips and selects them randomly to ask students questions, but the professor replaces the paper after calling on the student so they can be selected again at any time.
+Similar to the SRS design, the simple random sample with replacement (SRSWR) design randomly selects the sample from the entire sampling frame.  However, while SRS removes sampled units before selecting again, the SRSWR instead replaces each sampled unit before drawing again, so units can be selected more than once.
+  - **Requirements**: The sampling frame must include the entire population.
+  - **Advantages**: SRSWR requires no information about the units apart from contact information.
+  - **Disadvantages**: The sampling frame may not be available for the entire population. This design is not generally feasible for in-person data collection. Units can be selected more than once, resulting in a smaller realized sample size. For small populations, SRSWR has larger standard errors than SRS designs.
```

Comment:
Do we need to define what "smaller realized sample means"?

Also, the statement about small pops, does this meant that larger pops have smaller s.e. compare to SRS?  If so, should we add as an advantage?